superset/templates/superset/basic.html

```diff
@@ -1,124 +1,130 @@
-    favicon.href.startswith("http") else assets_prefix }}{{favicon.href}}" > {%
-    endfor %}
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/flags/flags16.css"
-    />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/fontawesome/fontawesome.min.css"
-    />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/fontawesome/regular.min.css"
-    />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/fontawesome/solid.min.css"
-    />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/fontawesome/brands.min.css"
-    />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="{{ assets_prefix }}/static/appbuilder/css/select2/select2.min.css"
-    />
+{# Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with this work
+  for additional information regarding copyright ownership. The ASF licenses this
+  file to you under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by
+  applicable law or agreed to in writing, software distributed under the License
+  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License. #}
 
-    {{ css_bundle("theme") }} {% if entry %} {{ css_bundle(entry) }} {% endif %}
-    {% endblock %} {{ js_bundle("theme") }}
+  <!DOCTYPE html>
+  {% import 'appbuilder/general/lib.html' as lib %} {% from
+  'superset/partials/asset_bundle.html' import css_bundle, js_bundle with context
+  %} {% set favicons = appbuilder.app.config['FAVICONS'] %}
+  <html>
+    <head>
+      {% include "head_custom_extra.html" %}
+      <title>
+        {% block title %} {% if title %} {{ title }} {% elif appbuilder and
+        appbuilder.app_name %} {{ appbuilder.app_name }} {% endif %} {% endblock
+        %}
+      </title>
+      {% block head_meta %}{% endblock %} {% block head_css %} {% for favicon in
+      favicons %} <link rel="{{favicon.rel if favicon.rel else "icon"}}"
+      type="{{favicon.type if favicon.type else "image/png"}}" {% if favicon.sizes
+      %}sizes={{favicon.sizes}}{% endif %} href="{{ "" if
+      favicon.href.startswith("http") else assets_prefix }}{{favicon.href}}" > {%
+      endfor %}
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="{{ assets_prefix }}/static/appbuilder/css/flags/flags16.css"
+        media="print"
+        onload="this.media='all'"
```

Comment:
I don't think this will work with prod CSP settings - violates no-eval rule

Reply:
Alright, makes sense. Let me see if we should handle this with a separate script.